### DISCUSSION TITLE
Formium's discord server fails to accept invites

### DISCUSSION THREAD

**Discussion by brandon-arnold** [jaredpalmer/razzle#1627] [Q&A] [answered]
Is Formium's Discord still the home of  #razzle-afterjs? It's referenced in the razzlejs.org and github README.md "Get Help" sections, but upon clicking it I'm getting a "Whoops...unable to accept invite" error.

**Comment by jaredpalmer** [comment, marked as answer]
Can you take a screenshot?

**Reply by brandon-arnold** [reply]
Uhhh. I must have rebooted since then and it seems to work now. Dang it! I mean, thanks. :)

https://user-images.githubusercontent.com/7827691/119279369-256e8300-bbe0-11eb-82aa-0f144a639f02.mp4
